vars/githubPr.groovy

```diff
@@ -106,23 +106,28 @@ def getTestFailuresMessage() {
   }
 
   def messages = []
+  messages << "---\n\n### [Test Failures](${env.BUILD_URL}testReport)"
 
-  failures.take(5).each { failure ->
+  failures.take(3).each { failure ->
     messages << """
----
-
-### [Test Failures](${env.BUILD_URL}testReport)
 <details><summary>${failure.fullDisplayName}</summary>
 
 [Link to Jenkins](${failure.url})
+"""
 
-```
-${failure.stdOut}
-```
-</details>
+    if (failure.stdOut) {
```

Comment:
Looks like different junit reports use these fields differently, so I need to check for all of them

Reply:
Will probably try to re-factor this stuff this week, the formatting/etc is really hard to follow because of needing to make everything look right in the PR comment